config/collectdata/vfb_fullontologies.txt

```diff
@@ -3,3 +3,4 @@ http://purl.obolibrary.org/obo/cl/cl-base.owl
 http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl
 http://purl.obolibrary.org/obo/hancestro/hancestro.owl
 https://www.ebi.ac.uk/efo/efo-base.owl
+http://purl.obolibrary.org/obo/mondo/mondo-base.owl
```

Comment:
I have added purl for mondo-base version. Whenever a new tag is added, we should also add purl of the required ontology. We should also have a documentation for this :)